app/components/embed/media_with_companion_windows_component.html.erb

```diff
@@ -76,7 +76,7 @@
             <h3>Media Content</h3>
           </div>
 
-          <ul class="contentsList" data-media-tag-target="list"></ul>
+          <ul class="contentsList" data-media-tag-target="list" role="tablist" data-controller="thumbnail"></ul>
```

Comment:
This data-controller should not be here. It's on the list items themselves.  You might consider adding a new controller for the list.

Reply:
Ah, gotcha. I had to move it up so that the targets were in scope but your suggestion below moves away from that. I'll give it a whirl. 👍